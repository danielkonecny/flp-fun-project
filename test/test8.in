A,B,C
a,b
A
A->aB
B->AAb
B->Ab
C->AbabA
C->bbb
