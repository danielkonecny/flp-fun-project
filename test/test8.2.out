<Ab>,<abA>,<bA>,<babA>,A,B,C,a',b',<bb>
a,b
A
<Ab>->Ab'
<abA>->a'<bA>
<bA>->b'A
<babA>->b'<abA>
A->a'B
B->A<Ab>
B->Ab'
C->A<babA>
C->b'<bb>
<bb>->b'b'
a'->a
b'->b
